proto/storage/cve.proto

```diff
@@ -88,6 +88,44 @@ message CVE {
     VulnerabilitySeverity       severity            = 20 [(gogoproto.moretags) = "search:\"Severity\""];
 }
 
+message CVEInfo {
+    message Reference {
```

Comment:
I guess we should keep this, but I think we can just deprecate it. It's never been widely used in our platform